### PR TITLE
Telcodocs 620 gpu console: QE edits

### PR DESCRIPTION
**Pull request by StephenJamesSmith**
TELCODOCS-620-GPU-CONSOLE - NVIDIA GPU administration dashboard

PR applies to the in-development version (e.g. 4.11) and future versions: 4.11+

Issue:
https://issues.redhat.com/browse/TELCODOCS-620?src=confmacro

Link to docs preview:
https://52233--docspreview.netlify.app/openshift-enterprise/latest/monitoring/nvidia-gpu-admin-dashboard.html

Additional information:

@fabiendupont - please review
@mareklibra

### CODE REVIEW

**Comment by StephenJamesSmith** [comment]
@wabouhamad @mresvanis I've created this new PR for TELCODOCS-620. It has all changes. Please review and comment with "/lgtm" if all of your changes are in.

File: modules/nvidia-gpu-admin-dashboard-installing.adoc [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+# if yes, then run the following command to enable the plugin
+$ oc patch consoles.operator.openshift.io cluster --patch '[{"op": "add", "path": "/spec/plugins/-", "value": "console-plugin-nvidia-gpu" }]' --type=json
+
+# add the required DCGM Exporter metrics ConfigMap to the existing NVIDIA operator ClusterPolicy CR:
+oc patch clusterpolicies.nvidia.com gpu-cluster-policy --patch '{ "spec": { "dcgmExporter": { "config": { "name": "console-plugin-nvidia-gpu" } } } }' --type=merge
+
+----
++
+The dashboard relies mostly on Prometheus metrics exposed by the NVIDIA DCGM Exporter, but the default exposed metrics are not enough for the dashboard to render the required gauges. Thus, the DGCM exporter should be configured to expose a custom set of metrics, as shown here.
```
Comment (GroceryBoyJr) [inline modules/nvidia-gpu-admin-dashboard-installing.adoc:76]: Recommend "therefore" vs "thus". Recommend "is" vs should be"

File: modules/nvidia-gpu-admin-dashboard-installing.adoc [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    app.kubernetes.io/part-of: console-plugin-nvidia-gpu
+    app.kubernetes.io/version: latest
+    helm.sh/chart: console-plugin-nvidia-gpu-0.2.3
+  name: console-plugin-nvidia-gpu
+  namespace: nvidia-gpu-operator
+  resourceVersion: "19096623"
+  uid: 96cdf700-dd27-437b-897d-5cbb1c255068
+----
++
+This is achieved by deploying this ConfigMap and editing the NVIDIA Operator ClusterPolicy CR to add that ConfigMap in the DCGM exporter configuration. The deployment of the ConfigMap is done by the new version of the Console Plugin NVIDIA GPU Helm Chart, but the ClusterPolicy CR editing should be done by the user.
```
Comment (GroceryBoyJr) [inline modules/nvidia-gpu-admin-dashboard-installing.adoc:114]: Recommend "installation" vs "deployment"

Comment (GroceryBoyJr) [inline modules/nvidia-gpu-admin-dashboard-installing.adoc:114]: Recommend replacing "should be done" passive voice with "is"

File: modules/nvidia-gpu-admin-dashboard-using.adoc [hunk 3]
```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * monitoring/nvidia-gpu-admin-dashboard.adoc
+
+:_content-type: PROCEDURE
+[id="nvidia-gpu-admin-dashboard-using_{context}"]
+= Using the NVIDIA GPU administration dashboard
+
+After deploying the the OpenShift Console NVIDIA GPU plugin, log in to the OpenShift Container Platform web console using your login credentials to access the *Administrator* perspective.
```
Comment (GroceryBoyJr) [inline modules/nvidia-gpu-admin-dashboard-using.adoc:9]: Recomend "plug-in" vs. "plugin"

**Review by GroceryBoyJr — COMMENTED** [review]
A few small nits, please.

File: modules/nvidia-gpu-admin-dashboard-introduction.adoc [hunk 4]
```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * monitoring/nvidia-gpu-admin-dashboard.adoc
+
+:_content-type: CONCEPT
+[id="nvidia-gpu-admin-dashboard-introduction_{context}"]
+= Introduction
+
+A new feature in this release is a dedicated administration dashboard for NVIDIA GPU usage visualization
```
[the remark this answers is not in the document]
Comment (StephenJamesSmith) [reply modules/nvidia-gpu-admin-dashboard-introduction.adoc:9]: Changed.

File: modules/nvidia-gpu-admin-dashboard-installing.adoc [hunk 5]
```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * monitoring/nvidia-gpu-admin-dashboard.adoc
+
+:_content-type: PROCEDURE
+[id="nvidia-gpu-admin-dashboard-installing_{context}"]
+= Installing the NVIDIA GPU administration dashboard
+
+Install the NVIDIA GPU plugin using Helm on the OpenShift Container
```
[the remark this answers is not in the document]
Comment (StephenJamesSmith) [reply modules/nvidia-gpu-admin-dashboard-installing.adoc:9]: Changed all occurrences except for those that are part of a URL, filepath, or code block. Remember this is NVIDIA source.

File: modules/nvidia-gpu-admin-dashboard-introduction.adoc [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+in the OpenShift Container Platform (OCP) Console. The visualizations in the administration dashboard provides guidance on how to
```
[the remark this answers is not in the document]
Comment (StephenJamesSmith) [reply modules/nvidia-gpu-admin-dashboard-introduction.adoc:10]: corrected.

File: modules/nvidia-gpu-admin-dashboard-installing.adoc [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+
+
+.Prerequisites
+
+* Red Hat OpenShift 4.11+
+* NVIDIA GPU operator
+* link:https://helm.sh/docs/intro/install/[Helm]
+
+
+.PROCEDURE
```
[the remark this answers is not in the document]
Comment (StephenJamesSmith) [reply modules/nvidia-gpu-admin-dashboard-installing.adoc:23]: corrected.

File: modules/nvidia-gpu-admin-dashboard-installing.adoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (StephenJamesSmith) [reply modules/nvidia-gpu-admin-dashboard-installing.adoc:9]: corrected.

File: modules/nvidia-gpu-admin-dashboard-installing.adoc [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+
+Use the following procedure to deploy the OpenShift Console NVIDIA GPU plugin.
```
[the remark this answers is not in the document]
Comment (StephenJamesSmith) [reply modules/nvidia-gpu-admin-dashboard-installing.adoc:25]: corrected.

File: modules/nvidia-gpu-admin-dashboard-installing.adoc (hunk 1 above)
Comment (StephenJamesSmith) [reply modules/nvidia-gpu-admin-dashboard-installing.adoc:76]: corrected.

File: modules/nvidia-gpu-admin-dashboard-installing.adoc (hunk 2 above)
Comment (StephenJamesSmith) [reply modules/nvidia-gpu-admin-dashboard-installing.adoc:114]: corrected.

[the remark this answers is not in the document]
Comment (StephenJamesSmith) [reply modules/nvidia-gpu-admin-dashboard-installing.adoc:114]: corrected.

Comment (StephenJamesSmith) [reply modules/nvidia-gpu-admin-dashboard-installing.adoc:114]: corrected.

File: modules/nvidia-gpu-admin-dashboard-using.adoc (hunk 3 above)
Comment (StephenJamesSmith) [reply modules/nvidia-gpu-admin-dashboard-using.adoc:9]: corrected.

**Pull request merged**